src/gui/processing/qgsprocessingaggregatewidgets.cpp

```diff
@@ -301,6 +302,10 @@ void QgsAggregateMappingModel::setMapping( const QList<QgsAggregateMappingModel:
 {
   beginResetModel();
   mMapping = mapping;
+  for ( auto &agg : mMapping )
```

Comment:
`as_const` or `mapping` (which already is const). You may also consider replacing `auto` with the data type